src/components/expired-subscription-modal/tests/ExpiredSubscriptionModal.test.jsx

```diff
@@ -1,14 +1,29 @@
 import { screen } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import userEvent from '@testing-library/user-event';
+import { AppContext } from '@edx/frontend-platform/react';
 import ExpiredSubscriptionModal from '../index';
-import { useSubscriptions } from '../../app/data';
+import { postUnlinkUserFromEnterprise, useEnterpriseCustomer, useSubscriptions } from '../../app/data';
 import { renderWithRouter } from '../../../utils/tests';
+import { authenticatedUserFactory, enterpriseCustomerFactory } from '../../app/data/services/data/__factories__';
 
 jest.mock('../../app/data', () => ({
   ...jest.requireActual('../../app/data'),
   useSubscriptions: jest.fn(),
+  useEnterpriseCustomer: jest.fn(),
+  postUnlinkUserFromEnterprise: jest.fn(),
 }));
+const mockAuthenticatedUser = authenticatedUserFactory();
+const mockEnterpriseCustomer = enterpriseCustomerFactory();
+
+const defaultAppContextValue = { authenticatedUser: mockAuthenticatedUser };
+const ExpirationModalWrapper = ({ children, appContextValue = defaultAppContextValue }) => (
```

Comment:
nit: to remove any confusion between this new component's name and the existing `SubscriptionExpirationModal` and the component under test (i.e., `ExpiredSubscriptionModal`), I might recommend renaming this component to `ExpiredSubscriptionModalWrapper`.

Reply:
make sense 👍